snapshots/src/hardened_unpack.rs

```diff
@@ -29,6 +29,8 @@ pub enum UnpackError {
     Io(#[from] std::io::Error),
     #[error("Archive error: {0}")]
     Archive(String),
+    #[error("Unpacking '{1}' failed: {0}")]
+    ArchiveFromPath(Box<UnpackError>, PathBuf),
```

Comment:
I'd suggest we rename this variant, since (1) the message doesn't mention "archive" anywhere, and (2) this variant is thrown from `unpack` fns.

```suggestion
    Unpack(Box<UnpackError>, PathBuf),
```

Reply:
done